src/DayColumn.js

```diff
@@ -188,7 +197,7 @@ class DayColumn extends React.Component {
       events,
       accessors,
       slotMetrics,
-      minimumStartDifference: Math.ceil(step * timeslots / 2),
+      minimumStartDifference: Math.ceil((step * timeslots) / 2),
```

Comment:
I don't think that this modification is part of the PR

Reply:
this is just prettier doing it's thing